sale_product_set_variants/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Sale Product Set Variants
+================
+
+Extension of the sale product set module to make it work with variants
+
+Usage
+=====
+
+* Activate the product.variants in sale -> settings
+TODO: complete usgae
```

Comment:
can you complete this?

Reply:
while you do this, please update the screenshots which are pretty old and show them here.